include/seqan3/io/alignment_file/output.hpp

```diff
@@ -228,6 +230,13 @@ class alignment_file_output
                   "please refer to the documentation of "
                   "seqan3::alignment_file_output::field_ids for the accepted values.");
 
+    static_assert([] () constexpr
+                  {
+                      return !(selected_field_ids::contains(field::ALIGNMENT) &&
+                               selected_field_ids::contains(field::CIGAR));
+                  }(),
+                  "You may not select field::ALIGNMENT and field::CIGAR at the same time.");
```

Comment:
actually why not? Both information are already present. And how do I specify to  get the cigar field instead of the alignment when I want otherwise have the default behavior? Is there a recommended way of doing it?

Reply:
hm.. I guess we could.

Reply:
ah no! I thought this is input.
But this is the output file. So of course we cannot provide both, the alingment and a cigar string because which should be written as they refer to the same thing.

Reply:
Ah is output! 👍